Example/Stripe iOS Example (Simple).xcodeproj/project.pbxproj

```diff
@@ -261,6 +261,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 8.4;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
+				OTHER_SWIFT_FLAGS = "-DDEBUG";
```

Comment:
Ah, a holdover from when I was running tests in the example app instead of the framework test target. I can remove.